users/templates/users/register/password_reset_confirm.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'users/base.html' %}
+{% load crispy_forms_tags %}
+{% block main %}
+   {% if validlink %}
+    <h3>Change password</h3>
+    <form method="post">
+      {% csrf_token %}
+      {{ form | crispy }}
+      <button type="submit">Change password</button>
+    </form>
+  {% else %}
+    <p>
+      The password reset link was invalid, possibly because it has already been used.
```

Comment:
<s>was</s> is.

If the link is already used then only you should show the error message that it's already used.

You shouldn't bring maybe/probably in error messages. The error message should be crisp and clear to the point which end user should understand without ambiguity.

Write a test case for this.